client/state/jetpack-connect/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { JETPACK_CONNECT_TTL } from './constants';
+
+/***
+ * Whether a Jetpack Connect store timestamp is stale.
+ * @param   {Number} timestamp  Item to check.
+ * @returns {Boolean}           True if the timestamp is stale, false otherwise.
+ */
+export function isStale( timestamp ) {
+	const now = new Date().getTime();
+	if ( ! timestamp ) {
```

Comment:
This may briefly cause some errors when we merge this for folks who are in the middle of the auth process.